src/integrators/sdirk_integrators.jl

```diff
@@ -224,15 +223,15 @@ end
     W = I - dto2*J
   else
     J = ForwardDiff.derivative(uf,uprev)
-    W = 1 - dto2*J
+    W = @. 1 - dto2*J
```

Comment:
Julia has been throwing a warning when using `I` like a number there since v0.5. It sucks so this is just a workaround to have it not throw the warning. The out-of-place Rosenbrock methods should be fixed to have the `I` route.